### DISCUSSION TITLE
API Visualizer

### DISCUSSION THREAD

**Discussion by zzFluke** [Kong/insomnia#5163] [Ideas]
One of the lacking features in Insomnia that makes me go to Postman is - https://www.postman.com/api-visualizer/. It does not have very great support and simplicity in terms of creation and maintenance of the custom visualizations, but it's definitely killer once you have it done and it works. It basically opens the world to small API-driven apps that can be prototyped and distributed together with the collection. As an example, APIs that I have been usually dealing with represents some sort of dependency ordered graph of objects returned in JSON, so I was able to build graph visualization and even manipulation using this neat Postman functionality that let me use 3rd party library like vizjs for the job. The other example is OData like APIs that return data in JSON and could be simply visualized with old-good jqgrid with it's sorting/searching/filtering capabilities.

What do you think about adding such functionality into Insomnia. I haven't dived deeper enough into the code, so it's hard to assess for me how invasive and complex implementation of such functionality could be.

**Comment by diegogurpegui** [comment]
+1 to this.
I've been using this in Postman and I somehow assumed it was present in Insomnia, but it's not. I didn't even find a plugin for this, is it? Or is it even possible to develop a plugin for doing this?
